processor/resourcedetectionprocessor/internal/resourcedetection_test.go

```diff
@@ -118,16 +118,35 @@ func TestDetectResource_Error(t *testing.T) {
 	require.EqualError(t, err, "err1")
 }
 
-func TestMergeOverride(t *testing.T) {
-	res1 := NewResource(map[string]interface{}{"a": "11", "b": "2"})
-	res2 := NewResource(map[string]interface{}{"a": "1", "c": "3"})
-	MergeResource(res1, res2, true)
-
-	expected := NewResource(map[string]interface{}{"a": "1", "b": "2", "c": "3"})
-
-	expected.Attributes().Sort()
-	res1.Attributes().Sort()
-	assert.Equal(t, expected, res1)
+func TestMergeResource(t *testing.T) {
+	for _, tt := range []struct {
```

Comment:
just curious what `tt` stands for

Reply:
It's usually used as the name when iterating over tests. Testing tables maybe?